### PR TITLE
latest changes for shipping and tax

### PR DESCRIPTION
**Pull request by devout27**

### CODE REVIEW

File: paysley/readme.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ Tags: credit card, paysley, google pay, apple pay, payment method, payment gatew
 Requires at least: 5.0
 Tested up to: 6.4.3
```
Comment (darren-paysley) [inline paysley/readme.txt]: @erwinesphere can we change this to latest wp version

Comment (erwinesphere) [reply paysley/readme.txt]: yes. we can. but it already tested or not?

Comment (darren-paysley) [reply paysley/readme.txt]: no its not apprently so we have to leave it as is till its teseted

File: paysley/paysley.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * Plugin Name:          Paysley
  * Plugin URI:           https://github.com/PaysleyLLC/paysley-woocommerce
  * Description:          Receive payments using Paysley.
- * Version:              2.0.1
+ * Version:              2.0.3
```
Comment (erwinesphere) [inline paysley/paysley.php:7]: should we change version to 2.0.2 or 2.0.3 ?

File: paysley/includes/class-paysley.php [hunk 3]
```diff
@@ -210,24 +210,25 @@ protected function get_payment_url($order_id, $transaction_id)
...
 			'payment_type' => $this->payment_type,
 			'request_methods' => ["WEB"],
 			'email' =>  $order->get_billing_email(),
-			// 'mobile_number' => $order->get_billing_phone(),
 			'mobile_number' => $customerPhoneNumber,
 			'customer_first_name' => $order->get_billing_first_name(),
 			'customer_last_name' => $order->get_billing_last_name(),
 			'currency'     => $currency,
 			'amount'       => (float) $amount,
+			'shipping_charges' => $order->get_shipping_total(),
```
Comment (erwinesphere) [inline paysley/includes/class-paysley.php:224]: so, shipping_charges = shipping amount + shipping tax?

**Pull request merged**